.kiro/specs/package-function-awareness/tasks.md

```
@@ -0,0 +1,304 @@
+    - _Requirements: 11.1, 11.2_
+
+  - [x] 13.2 Handle shadowing
+    - Navigate to local definition if it shadows package function
+    - _Requirements: 11.3_
+
+- [x] 14. Add Configuration Options
+  - [x] 14.1 Add package configuration to CrossFileConfig
+    - Add `packages_enabled: bool` (default: true)
+    - Add `packages_additional_library_paths: Vec<PathBuf>`
+    - Add `packages_r_path: Option<PathBuf>`
+`    - Add `packages_missing_package_severity: DiagnosticSeverity`
```

markdownlint-cli2 (0.20.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)